src/lib/components/admin/Settings/CodeExecution.svelte

```
@@ -0,0 +1,317 @@
+<script lang="ts">
+	import { toast } from 'svelte-sonner';
```

ESLint

[error] 'toast' is defined but never used.

(@typescript-eslint/no-unused-vars)